routes/services/study/getStudyList.js

```
@@ -0,0 +1,10 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+const getStudyList = async (req, res, next) => {
+    try {
+    }
+    catch (error) {
+        next(error);
+    }
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)